src/components/home.js

```diff
@@ -6,10 +6,20 @@ import { Link } from "react-router-dom";
 import { getArticlesAction } from "../actions/getArticles";
 
 export class Home extends Component {
+  constructor(props) {
+    super(props);
+    this.fetchNext = this.fetchNext.bind(this);
+    this.fetchPrevious = this.fetchPrevious.bind(this);
+  }
   componentDidMount() {
     this.props.getArticlesAction();
   }
-
+  fetchNext() {
+    this.props.getArticlesAction(this.props.next);
+  }
+  fetchPrevious() {
+    this.props.getArticlesAction(this.props.previous);
+  }
```

Comment:
Great work @e-ian . Kindly destructure the props in componentDidMount(), fetchNext() and fetchPrevious() methods. Otherwise swift implementation.

Reply:
Done with destructuring

Reply:
Great work @e-ian